exploration/demo/evaluate.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python3
+
+from pathlib import Path
+
+import polars as pl
+
+from linmod.eval.proportions import mae
+
+data = (
+    pl.scan_csv(Path("../../data/metadata.csv"))
+    .cast({"date": pl.Date}, strict=False)
+    .drop_nulls(subset=["date"])  # Drop dates that aren't resolved to the day
+    .filter(pl.col("date") >= pl.col("date").max() - 90)
```

Comment:
let's put this 90 as a global variable up top
```python
EVAL_PERIOD_DAYS = 90
```
or similar

will make it easier to refactor later

Reply:
I actually intend to move that entire filter statement into our global preprocessing script, since it happens every time I use the data anyway. Should I still make that change in this PR?

Reply:
my idea is more that "magic" numbers should be easy to find & identify. so it's less about where this bit is in the codebase, but more about how even the preprocessing scripts should have their parameters as easy to find & modify as possible (b/c I'm sure at some point we'll ask how the model performance would differ if we used a different evaluation period)

Reply:
Totally. I made the change in another (draft) PR that deals with more data-processing-oriented tasks, it looks like this:

https://github.com/CDCgov/cfa-viral-lineage-model/blob/8dd8b0ccbcec850e768a63b724fcb30d0263e865/linmod/data.py#L33

Does that look alright?

Reply:
nice, good enough for now